insights/tests/__init__.py

```diff
@@ -106,8 +115,7 @@ def run_test(component, input_data, expected=None):
     if filters.ENABLED:
         mod = component.__module__
         sup_mod = '.'.join(mod.split('.')[:-1])
-        rps = _get_registry_points(component)
-        filterable = set(d for d in rps if dr.get_delegate(d).filterable)
+        filterable = _get_registry_points(component, True)
```

Comment:
Let's revert this so that filtering to just the  filterable registry points happens here. It lets `_get_registry_points` have one job and makes it more explicit at this site that we only want the filterable ones. `get_registry_points(thing, True)` doesn't tell us much, and once we understand it, it raises the question of why there isn't an option for `raw` as well.

Reply:
Yes, this makes sense. I will change it back.